qemu/tests/hv_vapic_test.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import re
+import os
+import time
+
+from virttest import error_context
+from virttest import utils_disk
+from virttest import env_process
+from virttest import data_dir
+from avocado.utils import process
+from virttest.qemu_storage import QemuImg
+from provider.storage_benchmark import generate_instance
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test the hv_vapic flag improvement
+    1) Create tmpfs data disk in host
+    2) Mount&format the disk in guest, then prepare the fio test
+       environment
+    3) Boot the guest with all hv flags
+    4) Run fio test, record the result's bw value
+    5) Shutdown and boot the guest again without hv_vapic flag
+    6) Run fio test again, record the result's bw value
+    7) Calculate the improvement value of the 2 fio tests,
+       then check if it is obvious enough
+
+    param test: the test object
+    param params: the test params
+    param env: the test env object
+    """
+
+    def _create_tmpfs_data_disk():
+        """
+        Create a tmpfs data disk
+        """
+        logging.info("Create tmpfs data disk")
+        disk_name_key = params["disk_name_key"]
+        tmp_dir = data_dir.get_tmp_dir()
+        tmpfs_dir = os.path.join(tmp_dir, "tmpfs")
+        process.system("mkdir -p " + tmpfs_dir, shell=True)
```

Comment:
Please use `os.makedirs(tmpfs_dir) `instead of  `process.system("mkdir -p " + tmpfs_dir, shell=True)`